Cargo.toml

```
@@ -12,7 +12,7 @@ categories = ["algorithms", "no-std"]
 edition = "2018"
 
 [dependencies]
-crc-catalog = "1"
+crc-catalog = "1.1"
```

Should this be a carer requirement?
```suggestion
crc-catalog = "^1.1"
```

---

Any requirement without an operator is a caret requirement by default: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio So this is already a caret requirement. :)